### ISSUE TITLE
[UX] Remove front-end label "(not verified)" for anonymous users

### ISSUE THREAD

**Issue by findlabnet** [backdrop/backdrop-issues#3390]
## Describe your issue or idea
This label is not so informative as should and may produce headache for novice, what it is mean - "(not verified)" - for what, or by whom?
### Steps to reproduce (if reporting a bug)
This label appears next to unregistered user name in comment (if "Name" field is provided) or next to "Anonymous" word.

Same annoying label appears on "Recent log messages(dblog)" page for any 403/404 warning and so on, even for cron run records.
### Proposed solution
Remove this label.
File `core/modules/user/user.theme.inc` - just remove line 265: 
`$variables['extra'] = ' (' . t('not verified') . ')';`

In case if this is acceptable for all someone with hands on code can add this small fix to their PR.
Thanks.

**Comment by klonos** [comment]
@findlabnet that label is there to denote that the person that made the comment has claimed to be someone, but that has not been verified (a process that usually happens by registering with an email). It is used as a measure to prevent impersonation, and I think that this is pretty standard and not just a Drupal/Backdrop thing; I see it in many blogs built with other CMSes.

Imagine this scenario:

- There is forum topic requesting a new feature to be added, and we call for a vote by comments.
- A person called Bob who is not registered in the forum makes a comment voting the feature to not be implemented because he doesn't like it.
- Bob sees that no verification was required for him to comment/vote.
- Bob adds another vote-down comment, this time calls himself klonos
- Bob adds another vote-down comment, calling himself findlabnet
- Bob adds another vote-down comment, calling himself olafgrabienski 
...
- You @findlabnet are somehow notified that there is a vote-down comment on that forum thread.
- You are a registered member, so you comment in the forum topic saying "I actually like this feature - I don't know who voted down on my behalf"
- You also notify me and @olafgrabienski about this, so we also comment on the topic saying that we did not post the previous comments.

With the feature currently in place, all fake votes/comments added by Bob are marked as "not verified", while all our comments will not have that label. So people will be able to tell which comments/votes are by legitimate/registered users.

If that label was not added, how do you propose we solve this use case scenario @findlabnet ?

**Comment by findlabnet** [comment]
> This label appears next to unregistered user name in comment (if "Name" field is provided) or next to "Anonymous" word.

Case, when uid is 0 and it is not planned to be changed - anonymous comments is allowed by site owner.
Name provided by anonymous cannot have link to user profile/page/anything to autorize someone, so case of trolling is just visible.

Second thing: how to you plan autorize log records?

Third: why so many years we can see same question on D-related forums - 'How to remove "(not verified)" in Drupal'?
And yes, I know how to do it via `hook_preprocess_username` and I did it for years.

**Comment by klonos** [comment]
I see what you are saying @findlabnet, but there are valid reasons for keeping this feature in too. So the actual issue I see here is this:

As a site builder, how do I easily remove the "not verified" label.

I see two possible solutions to this problem:

1. Make this a `<span>` tag with a specific class, so that people can use CSS and easily hide this from display:

  `<span class="not-verified">(not verified)</span>`

2. Add a setting/checkbox for it somewhere, so that site builder can disable it completely, and not have it be rendered at all in the first place. If we decide to go with this, where should this setting live? ...should it be site-wide, or per content type? ...or per node?

> Second thing: how to you plan autorize log records?

I take it that you are annoyed by this being added to the log entries. Not sure if having it there or not makes sense. Can you please provide a few sample log entries where this is added vs some where it is not?

**Comment by findlabnet** [comment]
> As a site builder, how do I easily remove the "not verified" label.

Yes, @klonos your proposal already in place for Drupal 7/8: under "Global settings" (admin/appearance/settings) for all themes we have checkbox "User verification status in comments", but not in Backdrop.

About log entries.
This is as it now:
![1](https://user-images.githubusercontent.com/11479969/48886535-fe7b3e80-ee34-11e8-9020-aacd58726522.png)
And how can be:
![2](https://user-images.githubusercontent.com/11479969/48886558-1226a500-ee35-11e8-9486-0c6323fec590.png)

Please note: first type of user's label looks as in D7, second - as in D8 and you can check such logs respectively.

**Comment by klonos** [comment]
Yeah, I did some research on this and was about to respond. D7 had a "Settings" tab under "Appearance", with a "User verification status in comments" checkbox:

![screen shot 2018-11-22 at 7 06 13 pm](https://user-images.githubusercontent.com/2423362/48889457-192bd280-ee8a-11e8-8c70-d5d9231beb60.png)

The set of checkboxes was available under the global theme settings tab, as well as per-theme, in each individual theme's tab. In Backdrop, most of these checkboxes were moved:

- "Logo", "Site name" and "Site slogan became settings for the header block in layouts:

  ![screen shot 2018-11-22 at 7 12 31 pm](https://user-images.githubusercontent.com/2423362/48889671-bedf4180-ee8a-11e8-9822-b2b595e36f24.png)

- "User pictures in posts": not sure what happened to it. Was this ever actually used in D7 core themes?

- "User pictures in comments" became an option in the comment settings for each individual content type:

  ![screen shot 2018-11-22 at 7 16 32 pm](https://user-images.githubusercontent.com/2423362/48889905-6eb4af00-ee8b-11e8-8187-fadbbd3a2081.png)

- "User verification status in comments": gone (this is what this issue here is about)

- "Shortcut icon" has been removed. Not sure if there is a valid use case where it is imperative for the site to have no favicon.

- "Main menu", and "Secondary menu" became individual blocks that can be placed via layouts.

- "Logo settings" and "Shortcut icon settings" have been moved under Configuration >> System >> Site information:

  ![screen shot 2018-11-22 at 7 30 59 pm](https://user-images.githubusercontent.com/2423362/48890567-42019700-ee8d-11e8-8cb7-490c600185d4.png)

Anyway. If we are to revive the "User verification status in comments" setting, we would a place where it should live. Should it be a theme setting, or a comment setting?

I guess that implementing this per-theme does not make much sense, since the most common use case is for sites to be using a single front-end theme anyway. So this leaves us with the need for a place to have the global, site-wide setting. And frankly, I do not see why we should have an entire separate "Settings" tab under "Appearance" only for a single checkbox.

We could make this a setting in the comments section per-content type. This to me makes more sense, as this is a comment-related setting, but will people coming from D7 still be looking for it under theme settings? And what happens for sites upgraded from D7 (how do we migrate a per-theme setting to a pre-comment one)?

I would like to get more feedback on this before we action it.

**Comment by olafgrabienski** [comment]
> "User pictures in posts": not sure what happened to it.

Even if slightly off-topic: I guess it's this setting on the content type now:  
**Post > Configure > Display settings > Display the author picture**.

**Comment by olafgrabienski** [comment]
> We could make this a setting in the comments section per-content type (...) but will people coming from D7 still be looking for it under theme settings?

Probably, but that's comparable to the other settings which have been moved, so let's move also this one.

> And what happens for sites upgraded from D7 (how do we migrate a per-theme setting to a per-comment one)?

Maybe: If "User verification status" was enabled on the default theme, enable it for comments of all content types. Else: Disable it for comments of all content types.

**Comment by klonos** [comment]
>> "User pictures in posts": not sure what happened to it.
>
>Even if slightly off-topic: I guess it's this setting on the content type now:
Post > Configure > Display settings > Display the author picture

Good catch @olafgrabienski 👍 that's it! I've updated my comment above.

>Maybe: If "User verification status" was enabled on the default theme, enable it for comments of all content types. Else: Disable it for comments of all content types.

That could work.

**Comment by BWPanda** [comment]
'Not verified' is not the best wording IMO. It's not immediately obvious what this means and what it's for (@klonos' [comment](https://github.com/backdrop/backdrop-issues/issues/3390#issuecomment-440775896) above was the first I'd heard about why this exists).

I think 'Guest' would be a more obvious term to use in this situation (meaning someone without an account). But if Drupal has solved this by just allowing it to be turned off, then maybe re-wording it in Backdrop would be too confusing...?

In any case, in addition to the above suggestions, I think we should make this text have a dashed underline which, when hovered over, displays some text explaining what this means. For example:

![image](https://user-images.githubusercontent.com/2385329/95044672-bfee8480-072b-11eb-84c2-866f7b5c19ca.png)
(pretend the red thing is the mouse cursor (which wasn't captured in the screenshot))
